### PR TITLE
Add observation to check if is in viewport

### PR DESCRIPTION
**Pull request by acardinale**
Add observation to check if is in viewport

### CODE REVIEW

**Comment by zhouyx** [comment]
to @ampproject/wg-ads

File: ads/nativery.js [hunk 1]
```diff
@@ -40,6 +41,19 @@ export function nativery(global, data) {
     }
   });
 
+  // install observation to check if is in viewport
+  var unlisten = global.context.observeIntersection(function(changes) {
+    changes.forEach(function(c) {
+      global._nativery.visible = parseInt(
```
Comment (powerivq) [inline ads/nativery.js]: Why using `parseInt` when the argument is not a string? Perhaps you want to do rounding down?

File: ads/nativery.js [hunk 2]
```diff
@@ -40,6 +41,19 @@ export function nativery(global, data) {
...
+        (c.intersectionRect.height / c.boundingClientRect.height) * 100,
+        10
+      );
+      if(global._nativery.visible) {
```
Comment (powerivq) [inline ads/nativery.js]: Nit: space `if (global`

File: ads/nativery.js (hunk 1 above)
Comment (acardinale) [reply ads/nativery.js]: Argument is float. Keep parseInt is a problem?

File: ads/nativery.js (hunk 2 above)
Comment (acardinale) [reply ads/nativery.js]: Resolved

File: ads/nativery.js (hunk 1 above)
Comment (powerivq) [reply ads/nativery.js]: `Math.floor` has been widely supported (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor), and seems to fit better to the scene.

Comment (acardinale) [reply ads/nativery.js]: Resolved

**Comment by mrjoro** [comment]
@acardinale sorry this is taking a while to get in.

Can you resolve the conflict and then confirm in a comment that it's still ready to merge?

**Pull request closed without merging**